brainframe/api/stubs/capsules.py

```diff
@@ -28,6 +29,38 @@ def get_capsules(self, timeout=DEFAULT_TIMEOUT) -> List[Capsule]:
         capsules, _ = self._get_json(req, timeout)
         return [Capsule.from_dict(d) for d in capsules]
 
+    def set_capsule(self, storage_id: int,
```

Comment:
Would it be a good idea to create a convenience method that's in charge of both upload the capsule data to storage, _and_ loading it into the server?

Reply:
Yeah, that's a good idea!